src/main/java/frc/robot/sensors/gyroSensor/GyroConstants.java

```diff
@@ -0,0 +1,11 @@
+package frc.robot.sensors.gyroSensor;
+
+public class GyroConstants {
+  // Port of the gyroscope //
+  public static final int gyroSensor = 0;
```

Comment:
probably better named something like gyroPort, or even just portNumber.  Same with renaming gyroFlipped to just flipped.  We know from the class it is part of that these parameters deal with gyros.